src/assets/javascript/components/bottomElements.js

```diff
@@ -0,0 +1,6 @@
+function bottomElements(textContent) {
```

Comment:
Change to button

Reply:
this one was just created to test all the buttons in a more separate way

Reply:
you shouldn't add more code, just to test!